src/main/java/club/sk1er/hytilities/config/HytilitiesConfig.java

```diff
@@ -120,6 +120,13 @@
     )
     public static boolean padPlayerCount;
 
+    @Property(
+            type = PropertyType.SWITCH, name = "Short Channel Names",
+            description = "Shortens channel names; Guild -> G , Party -> P",
```

Comment:
maybe keep this inline with how we usually do examples, `§eExample: Guild -> G, Party -> P`